recipes/excitingtools/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "excitingtools" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/excitingtools-{{ version }}.tar.gz
+  sha256: cd1f6cf62e7e4b6a9b1d1de2fd26e0d9e3e032f5082c02be536aef5be3ae3dd2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools >=35.0.2
+    - setuptools-scm >=2.0.0,<3
```

Comment:
Any reason for pinning `setuptools-scm` to such an old version (2018)?
First version available on conda-forge is 3.5.0. Current version is 7.1.0.

Reply:
An oversight on my part when moving from `setup.py` to `pyproject.toml` (I copy-pasted `[build-system] `from an example).

Updated to `setuptools-scm >= 3.5.0` in the recipe and the `excitingtools` project.